examples/layers/macos/Runner/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
I think we usually leave them out and just let the app icon be blank, but I'm not sure if that's the right thing to do for an "example" of how to build a mac app.

Reply:
I'd remembered something along the same lines; or that we'd checked them into a separate assets repo, but noticed the iOS equivalent has them checked in, so had left them. 

I'm fine blowing them away here; the layers example isn't exactly a great starting point for anyone looking at how to write an app, since it's got heaps of main entry points and a bit of a strange layout.

Reply:
If you don't care I guess prefer not checking new binaries into the repo.